pkg/discovery/probe/node_probe.go

```diff
@@ -196,11 +208,26 @@ func (nodeProbe *NodeProbe) createCommoditySold(node *api.Node, nodePodsMap map[
 		}
 	}
 
-	// Use Kubernetes service UID as the key for cluster commodity
+	// Create a new VMPMAccessCommodity to indicate if the node is schedulable. If the Node is schedulable, then
+	// the node will sell it. Otherwise, it doesn't sell. On the consumer side, if a pod can be scheduled by
```

Comment:
yes, as Cheuk explained, if a pod cannot be scheduled by scheduler, we set it to not monitored in pod probe. 

Here I think the main purpose of this accessCommodity is to prevent pods can be scheduled from being scheduled to unschedulable nodes. we can think this schedulable status of a node like a node label, and pods want to be scheduled should select nodes with this schedulable label. Then the way we handle it is just like any other label-selector. 

As for other pods, such as mirror pods or created by DaemonSets, I think how do we control their behavior is a different problem. As Enlin mentioned, they don't buy this schedulable accessCommodity, so from our market stand point, they can run on any node, either schedulable or unschedulable. This is what we expected. For these pods, we only want to monitor them, and prevent them from moving, scaling, etc, basically exclude them from shopping around. We set them to not monitored to make market unaware of these pods.